src/main/java/com/hazelcast/gcp/GcpMetadataApi.java

```diff
@@ -61,7 +66,15 @@ String accessToken() {
     }
 
     private static String extractAccessToken(String accessTokenResponse) {
-        return Json.parse(accessTokenResponse).asObject().get("access_token").asString();
+        try {
+            return Json.parse(accessTokenResponse).asObject().get("access_token").asString();
+        } catch (ParseException e) {
+            LOGGER.severe("Unable to retrieve access token. "
```

Comment:
Change message to something like: `Unable to retrieve access token. Please grant permissions to this service account if running from within the GCP network or specify the correct private key file path if running from outside the GCP.`